hw/ip/prim/dv/prim_alert/prim_alert_sim_cfg.hjson

```diff
@@ -45,18 +45,22 @@
   tests: [
     {
       name: prim_async_alert
+      run_opts: ["+test_timeout_ns=1_000"]
```

Comment:
where do we use this plusarg? We have it in dv_base_test, but seems like we don't use dv_base_test here